### PR TITLE
[Service] Scaffold the AddService command and nothing else

### PR DESCRIPTION
**Pull request by h5law**
### Human Summary

- First ran
```sh
ignite scaffold message add-service --module service --signer address --yes
```
- Moved some files around to make the blockchain build
- Removed double import from scaffold

### Summary generated by Reviewpad on 12 Jan 24 16:40 UTC
This pull request includes the following changes:

1. The file "codec.go" underwent several changes, including the addition of registration for the `MsgAddService` concrete type in the `RegisterCodec` function, the `RegisterInterfaces` function, and the `_Msg_serviceDesc` in the `RegisterInterfaces` function.
2. A new test file named `message_add_service_test.go` was added in the `types` package. It includes a test function called `TestMsgAddService_ValidateBasic` that tests the `ValidateBasic` method of the `MsgAddService` struct using various test cases.
3. There were changes to the file dependencies. The dependencies "github.com/cosmos/cosmos-proto" and "google.golang.org/genproto/googleapis/api" were added and then later removed.
4. The file `CmdAddService` was added to the service client CLI, which defines a command for broadcasting the `add-service` message and includes transaction-related options.
5. A new file named `message_add_service.go` was added to the `types` package, containing the implementation of the `MsgAddService` struct and its associated methods, as well as basic validation logic.
6. Changes were made to a file that added a new `AddService` RPC method and two new message types, `MsgAddService` and `MsgAddServiceResponse`.
7. A new file named `add_service.go` was added to the `x/service/simulation` directory, which contains simulation logic for the `SimulateMsgAddService` function.
8. The `tx.go` file added the `CmdAddService()` command to the existing `GetTxCmd()` command.
9. Changes were made to the file `msg_server_test.go`, including import statements and reorganization of code.
10. A new file named `msg_server_add_service.go` was added to the `x/service/keeper` package, which implements the `AddService` method but currently lacks logic and returns an empty response.
11. Changes were made to the file `module_simulation.go`, including the addition of constants and modification of functions related to simulating the `SimulateMsgAddService` message.

Please review these changes and ensure they align with the intended modifications for this pull request.

## Issue

- Fixes (in-part) [#230](https://github.com/pokt-network/poktroll/issues/230)

Select one or more:

- [x] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [x] **Run all unit tests**: `make go_develop_and_test`
- [x] **Run E2E tests locally**: `make test_e2e`
- [ ] **Run E2E tests on DevNet**: Add the `devnet-test-e2e` label to the PR. This is VERY expensive, only do it after all the reviews are complete.

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [x] I have performed a self-review of my own code
- [x] I have commented my code, updated documentation and left TODOs throughout the codebase

### CODE REVIEW

File: x/service/types/message_add_service_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/pokt-network/poktroll/testutil/sample"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMsgAddService_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  MsgAddService
+		err  error
```
Comment (Olshansk) [inline x/service/types/message_add_service_test.go:15]:
```suggestion
		desc string

		msg  MsgAddService

		expectedErr  error
```

File: x/service/client/cli/tx_add_service.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package cli
```
Comment (Olshansk) [inline x/service/client/cli/tx_add_service.go:1]: Reminder to add `add_service_test.go` when we implement

File: proto/pocket/service/tx.proto [hunk 3]
```diff
@@ -1,7 +1,16 @@
...
 
 option go_package = "github.com/pokt-network/poktroll/x/service/types";
 
 // Msg defines the Msg service.
-service Msg {}
\ No newline at end of file
+service Msg {
+  rpc AddService (MsgAddService) returns (MsgAddServiceResponse);
+}
+message MsgAddService {
```
Comment (Olshansk) [inline proto/pocket/service/tx.proto:11]: Reminder add the proper tags when we add signers.

```suggestion

message MsgAddService {
```

**Review by Olshansk — APPROVED** [review]
Leaving comments for the follow up (not this PR) as a reminder. Feel free to merge and solve then.

File: x/service/types/message_add_service_test.go (hunk 1 above)
Comment (h5law) [reply x/service/types/message_add_service_test.go:15]: These are changed in followup PR

**Pull request merged**